src/migrations/1725962477957-initial-schema.ts

```diff
@@ -0,0 +1,127 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
```

Comment:
This file was 100% generated by `npm run migration:generate` and then just auto cleaned up with prettier and eslint.

Happy to add whitespace to make it more human-readable if you prefer?

Reply:
This should be fine.  I don't imagine us looking to closely at it as its generated by TypeOrm unless something goes wrong.